queries.sh

```diff
@@ -126,4 +126,7 @@ else
     psql -h $HOST -p $PORT -d $DATABASE --user=$USER -A -F ',' -f $INPUT -o $OUTPUT $TUPLE_ONLY
 fi
 
+# Remove line counter from end of CSV
+sed -i '' -e '$ d' $OUTPUT
```

Comment:
As referenced here (http://stackoverflow.com/questions/4881930/bash-remove-the-last-line-from-a-file):

On Mac OS X (as of 10.7.4), the equivalent of the sed -i command above is sed -i '' -e '$ d' foo.txt



> On 6 Apr 2015, at 15:05, Vincent Delorme <notifications@github.com> wrote:
> 
> In queries.sh:
> 
> > @@ -126,4 +126,7 @@ else
> >      psql -h $HOST -p $PORT -d $DATABASE --user=$USER -A -F ',' -f $INPUT -o $OUTPUT $TUPLE_ONLY
> >  fi
> >  
> > +# Remove line counter from end of CSV
> > +sed -i '' -e '$ d' $OUTPUT
> Nice! 
> Is this any different from sed -i '$ d' $OUTPUT ?
> 
> —
> Reply to this email directly or view it on GitHub.
>